src/Microsoft.Management.Configuration.Processor/DSCv3/Unit/DSCv3ConfigurationUnitProcessor.cs

```diff
@@ -54,5 +57,49 @@ protected override bool ApplySettingsInternal()
         {
             return this.processorSettings.DSCv3.SetResourceSettings(this.UnitInternal, this).RebootRequired;
         }
+
+        /// <inheritdoc />
+        protected override IList<ValueSet>? GetAllSettingsInternal()

```

Comment:
From reading the caller implementation, this GetAllSettingsInternal() is for completeness, and the ConfigurationProcessor implementation will always go to GetAllUnitsInternal()?

Reply:
If the caller invokes `GetAllSettings`, this will still be called.  If we want to deprecate `GetAllSettings` though, this code could be migrated to the `ConfigurationProcessor` implementation and it could be updated to use the `GetAllUnits` interface as well.